quipucords/tests/api/source/test_source.py

```diff
@@ -1595,3 +1601,87 @@ def test_update_openshift_range_hosts(self):
         assert response.status_code == status.HTTP_400_BAD_REQUEST
         response = response.json()
         assert response["hosts"][0] == messages.SOURCE_ONE_HOST
+
+    def test_acs_source_create(self):
+        """Ensure we can create a new ACS source."""
+        data = {
+            "name": "acs_source_1",
+            "source_type": DataSources.ACS,
+            "hosts": ["1.2.3.4"],
+            "credentials": [self.acs_cred_id],
+        }
+        self.create_expect_201(data)
+        assert Source.objects.count() == 1
+        assert Source.objects.get().name == "acs_source_1"
```

Comment:
pls don't create new tests using unittest.

also, since you are touching `test_sources` you are now eligible to convert this whole file. yay

Reply:
(this comment applies to all tests in this file)